css/css-values/support/mixed-units-07.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <style>
+      body {
+        margin: 0;
+        font-size: 16px;
```

Comment:
This is misleading. Some queries query need `1em = 16px` indeed, but this declaration has no effect on that, as per https://drafts.csswg.org/mediaqueries-5/#units

So I would just remove `font-size: 16px`, and then just rely on https://web-platform-tests.org/writing-tests/assumptions.html?highlight=16px#test-assumptions

> The `medium` `font-size` computes to 16px.

Reply:
@Loirooriol I was matching the values set in previous neighboring tests. Should I remove those declarations for all of them?
* [mixed-units-01.html](https://github.com/web-platform-tests/wpt/blob/master/css/css-values/support/mixed-units-01.html)
* [mixed-units-02.html](https://github.com/web-platform-tests/wpt/blob/master/css/css-values/support/mixed-units-02.html)
* [mixed-units-03.html](https://github.com/web-platform-tests/wpt/blob/master/css/css-values/support/mixed-units-03.html)
* [mixed-units-04.html](https://github.com/web-platform-tests/wpt/blob/master/css/css-values/support/mixed-units-04.html)
* [mixed-units-05.html](https://github.com/web-platform-tests/wpt/blob/master/css/css-values/support/mixed-units-05.html)
* [mixed-units-06.html](https://github.com/web-platform-tests/wpt/blob/master/css/css-values/support/mixed-units-06.html)

Reply:
Whatever you prefer. I can change the existing tests in a follow-up if you only want to add new tests here.

Reply:
@Loirooriol So when you say to rely on https://web-platform-tests.org/writing-tests/assumptions.html?highlight=16px#test-assumptions, are you suggesting that I use `font-size: medium` someplace, or that I simply allow the browser to set the 16px by default.

Just want to make sure I'm clear on what to do about the font-sizing.

Reply:
@Loirooriol Could you check my latest work in commit [9c784ec](https://github.com/web-platform-tests/wpt/pull/42517/commits/9c784ec62bf53ecd2f38483e145342142cf1e7e8)

I completely removed the `font-size` style declaration

Reply:
Yes, just remove it, since it has no effect. In a media query, `em` units will resolve using the initial value of `font-size`, which is `medium`, and the docs say we can assume that it computes to 16px.